### DISCUSSION TITLE
custom domain created through vercel, shows domain registered under third-party, unable to edit nameserver.

### DISCUSSION THREAD

**Discussion by ghost** [vercel/community#7119] [Help] [answered]
### Summary

I created a _custom domain_ through **vercel**, now the problem is when I tried to point my vercel domain to an external DNS provider ( namely **cloudflare**) it asked me to change the nameserver in vercel, but when I came to change the name server in vercel it's showing it's registered under third party and I can't make changes to nameserver , while originally I got the custom domain through vercel itself so I can't figure out how it can be registered under third part, which is preventing me from changing my _**name server**_ through vercel and pointing to my third party DNS provider.

![Screenshot 2024-06-07 101457](https://github.com/vercel/community/assets/102549463/aef66f11-10c3-4535-bf44-29d665b9a35c)
![Screenshot 2024-06-07 101532](https://github.com/vercel/community/assets/102549463/55672f97-7f92-4936-a416-4cac44b46448)

I'm a newbie to web hosting and everything related and might be facing a dumb problem but I've tried finding a solution through the guides available but couldn't find my particular problem as it disables me from editing the nameserver itself.

approaches I have tried:
1. creating other domains through vercel but all are registered under third party.
2. changing TLD ( I thought it might be the problem at first but turned out it's not irrespective of the TLD or how many times I delete and recreate my domains I can't edit their nameserver)
3. since it's showing registered under third-party I tried finding my domain through WHOIS but it shows my domain is unregistered.

![image](https://github.com/vercel/community/assets/102549463/fcadebe8-3dec-4e21-baee-c1eae941dbeb)

At this point I'm out of ideas.

### Steps to Reproduce

1, create a custom domain through vercel
2. edit the name server to point towards an external DNS provider.

**Comment by amyegan** [comment, marked as answer]
Hey @Mebistieux. It looks like that domain was added to your project, but was not purchased or configured to work on your Vercel account. To use a custom domain you need to purchase it. This can be done on your Vercel account or through any domain registrar.  

- [Add a domain](https://vercel.com/docs/getting-started-with-vercel/domains#add-a-domain)
  - [Buy a domain with Vercel](https://vercel.com/docs/getting-started-with-vercel/buy-domain#buy-a-domain)
  - [Use a domain owned somewhere else](https://vercel.com/docs/getting-started-with-vercel/use-existing#use-an-existing-domain)

**Reply by ghost** [reply]
Thank you, I've understood what I was doing wrong now.
